dashboard/dashboard/views.py

```diff
@@ -125,6 +126,15 @@ def view_administration(request):
         context['plugins'] = plugin_client.get_plugins_info()
         context['job_type_versions'] = plugin_client.get_job_type_versions()
         context['infrastructure_targets'] = plugin_client.get_infrastructure_targets()
+
+        #Collect instances running on the same infrastructure
+        _infrastructure_instances: dict = collections.defaultdict(list)
+        for instance, infrastructure in context['infrastructure_targets'].items():
```

Comment:
In fact, there's `infrastructure_name, plugin_manifest` inside this dict and I think it's more accurate to call them that way instead of `instance, infrastructure`